obelix/panoramix/main.go

```diff
@@ -1,9 +1,34 @@
 package main
 
 import (
-	"fmt"
+	"context"
+	"github.com/PtitLuca/go-dispatcher/dispatcher"
+	"log"
+	"panoramix/cli"
+	"panoramix/configuration"
+	"panoramix/services/google"
 )
 
 func main() {
-	fmt.Println("Begin panoramix")
-}
\ No newline at end of file
+	ctx := context.Background()
+	cliFlags := cli.Parse()
+
+	conf, err := configuration.ExtractConfiguration(cliFlags.ConfigurationPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	gclient, err := google.New(ctx, conf.GoogleScopes)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	d := dispatcher.New()
+	if err := d.Register("google", gclient); err != nil {
+		log.Fatalln(err)
+	}
+
+	if _, err := d.Run("google", "CreateNewDocument", "MyDocumentTitle"); err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
You could create an `init` function to register service in their own package instead of everything in main

Reply:
Added in https://github.com/Cleopha/ifttt-like/pull/28/commits/9d48e364b3b5d4e10ecdf48d34e1311079674662